src/HttpMethodsClient.php

```diff
@@ -184,7 +207,7 @@ public function options($uri, array $headers = [], $body = null)
      */
     protected function send($method, $uri, array $headers = [], $body = null)
     {
-        return $this->httpClient->sendRequest($this->messageFactory->createRequest(
+        return $this->sendRequest($this->messageFactory->createRequest(
```

Comment:
i guess we should do this, in case somebody extends the methods-client to add something on sendRequest